lib/auth-middleware/auth_middleware/__init__.py

```diff
@@ -97,17 +97,30 @@ def authorize(group_name=None):
         id_info = g.user_token_data
```

Comment:
Could you please also update the group names at the top of this file? It would help for the upcoming changes. Thanks.
```
rokwire_event_manager_group = 'urn:mace:uiuc.edu:urbana:authman:app-rokwire-service-policy-rokwire events manager'
rokwire_events_uploader = 'urn:mace:uiuc.edu:urbana:authman:app-rokwire-service-policy-rokwire ems events uploader'
rokwire_events_web_app = 'urn:mace:uiuc.edu:urbana:authman:app-rokwire-service-policy-rokwire events web app'
rokwire_events_approvers = 'urn:mace:uiuc.edu:urbana:authman:app-rokwire-service-policy-rokwire event approvers'
rokwire_app_config_manager_group = 'urn:mace:uiuc.edu:urbana:authman:app-rokwire-service-policy-rokwire app config manager'

ROKWIRE_EVENT_WRITE_GROUPS = [rokwire_event_manager_group, rokwire_events_uploader, rokwire_events_web_app,
                              rokwire_events_approvers]
ROKWIRE_APP_CONFIG_WRITE_GROUPS = [rokwire_app_config_manager_group]
```

Reply:
Added